services/movie_session.py

```diff
@@ -13,6 +13,13 @@ def create_movie_session(
     )
 
 
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    return list(Ticket.objects.filter(
+        movie_session_id=movie_session_id).values(
+        "row", "seat")
```

Comment:
In case there are no arguments on the same line with an opening parenthesis, your closing parenthesis should not have any arguments on the same line as well. Please, resolve this issue